src/components/Table/TableHead/tools/cells.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Did we mean to bring all of these extra files from Carbon?

Reply:
Now removed, after using const sortStates = {
NONE: 'NONE',
DESC: 'DESC',
ASC: 'ASC',
};